src/get-node-text.js

```diff
@@ -1,15 +1,11 @@
+const TEXT_NODE = 3
 function getNodeText(node) {
-  const window = node.ownerDocument.defaultView
-
   if (node.matches('input[type=submit], input[type=button]')) {
     return node.value
   }
 
   return Array.from(node.childNodes)
-    .filter(
-      child =>
-        child.nodeType === window.Node.TEXT_NODE && Boolean(child.textContent),
-    )
+    .filter(child => child.nodeType === TEXT_NODE && Boolean(child.textContent))
```

Comment:
Here I opted to just inline the TEXT_NODE constant. Alternatively it looks like this is also available as `document.TEXT_NODE` although I'm not entirely sure what the browser support for that is. Using the constant seemed sane as it is only in one place so far.